src/hooks/cart.js

```diff
@@ -13,8 +13,11 @@ export default class extends BaseHooks {
     }
 
     itemAdd() {
-        this.$body.on('submit', '[data-cart-item-add]', (event) => {
```

Comment:
This is an attribute selector that matches elements like `<a data-cart-item-add href="http://foobar.com">foo</a>`. Your replacement using `querySelector('data-cart-item-add')` will not match the same thing. It will match elements like `<data-cart-item-add foo="bar"/>`.

Reply:
fixed typo.